controllers/common_test.go

```diff
@@ -148,4 +158,42 @@ var _ = Describe("GetGeneratedJobName", func() {
 			Expect(uidWithoutHyphens).To(ContainSubstring(generatedJobName))
 		})
 	})
+
+	When("An additional postfix is required and maxNameLen is sufficiently large", func() {
+		BeforeEach(func() {
+			maxNameLen = 253
+			objectMetaName = "object.meta.name"
+			generatedPostfix = "generated.postfix"
```

Comment:
This usage is confusing to me. Who generates this postfix? What is it postfixed to?

Reply:
I think these tests would be much more clear if they used `required` and `optional` strings, following the parameters of `GetGeneratedResourceName`.

Reply:
did not update the tests very well to match the changed methods, fixing